internal/deviations/deviations.go

```diff
@@ -486,6 +486,25 @@ func StorageComponentUnsupported(_ *ondatra.DUTDevice) bool {
 	return *storageComponentUnsupported
 }
 
+// PlatformPowerEnableWait returns timer value to wait while platform is powered on.
+// Default value is 0.
+func PlatformPowerEnableWait(dut *ondatra.DUTDevice) int64 {
+	logErrorIfFlagSet("deviation_platform_power_enable_wait")
+	ppewait := lookupDUTDeviations(dut).GetPlatformPowerEnableWait()
+	var min_ppwewait int64 = 5
```

Comment:
Does this mean that we do not want this proto field to be less than 5 ? 
If yes, why? Is 0 an invalid value?

Reply:
Updated logic